add-timestamp.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e -u
+
+function set_timestamp() {
+    directory=$1;
+    last_modified=$(git log -n 1 --pretty='format:%cd' --date=format:'%d.%m.%Y' "$1")
```

Comment:
@redfast00 I would use `%B %e %Y` as formats which yields `September  6 2018`, rather than `06.09.2018` which leaves it open to interpretation (is it June 9th or Sept 6th?).